src/utils/package.ts

```diff
@@ -0,0 +1,17 @@
+import * as t from "io-ts";
+// tslint:disable-next-line: no-var-requires
+const packageJson = require("../../package.json");
```

Comment:
can you check if this works when the app gets transpiled to js (i.e. in the docker image)?